ecosystem.config.cjs

```diff
@@ -0,0 +1,13 @@
+module.exports = [
+	{
+		script: 'cli.js',
+		name: 'directus',
+		exec_mode: 'cluster',
+		instances: 0,
+		args: ['start'],
+		wait_ready: true,
```

Comment:
What will happen in case of K8S healthchecks while app not started yet?

Reply:
Would it introduce any difference? 🤔

Reply:
Should be the exact same